src/pc/mod.rs

```diff
@@ -68,7 +69,7 @@ pub(crate) struct ParachainCommanderOptions {
 	pub node: String,
 	/// Parachain id.
 	#[clap(long)]
-	para_id: u32,
+	para_id: Vec<u32>,
```

Comment:
From a UX perspective this doesn't seem easy to use. If we deploy on Versi with 70 parachains, it will be quite a long cmd line. We could make this field optional and if it is not specified we track all parachains that get a core assigned. This would enable us to also track parathreads without any other change.

I am hoping implementing this is easier after the collector and pc refactor. 

As an implementation I suggest to have a task that takes care of spawning and stopping instances of `watch_node`. The task would also need to keep track of the [core assignments](https://github.com/paritytech/polkadot/blob/dfa8c5ae321e0523f88fdc43afc6d743812ee49e/runtime/parachains/src/scheduler.rs#L123) and perform some pruning for parachains that didn't get a core assigned to them for a period of time.

Reply:
We can probably add a special parahain id `all` to subscribe to broadcast updates from the collector. However, that will require some major changes in the logic of the tracker and statistics to include per-parachain filter for everything.

Reply:
However, this task is really a separate one: with this change we can easily take some parachains and observe them in the real-time simultaneously comparing them block-by-block. A take-them-all mode is definitely useful but it should probably use a different subscription mode.